### PR TITLE
Update privacy notice

### PR DESCRIPTION
**Pull request by NoraGDS**
Ready for review

---

Preview: https://prototype-kit-docs-pr-44.herokuapp.com/docs/privacy-policy

### CODE REVIEW

File: docs/views/privacy-policy.html [hunk 1]
```diff
@@ -20,60 +20,154 @@
...
+  
+  <ul class="govuk-list govuk-list--bullet">
+      <li>software collaboration platforms when you share research, feedback or make a contribution</li>
+      <li>mailing list providers when you sign up to receive emails from us</li>
+      <li> <a href="https://www.gov.uk/government/publications/government-digital-service-user-support-privacy-notice/user-support-privacy-notice">support providers</a> when you contact us for assistance
+      </li>
+      <li>web analytics services</li>
+    </ul>
+   
+   <p>We share a mailing list provider with the GOV.UK Design System. This means your data is part of the same system as the GOV.UK Design System mailing list.
```
Comment (lfdebrux) [inline docs/views/privacy-policy.html]:
```suggestion
   <p>We share a mailing list provider with the GOV.UK Design System. This means your data is part of the same system as the GOV.UK Design System mailing list.</p>
```

File: docs/views/privacy-policy.html [hunk 2]
```diff
@@ -20,60 +20,154 @@
...
+   <p>The Data Protection Officer provides independent advice and monitoring of Cabinet Office’s use of personal information.</p>
+   
+   <p>If you consider that your personal data has been misused or mishandled, you may make a complaint to the Information Commissioner, who is an independent regulator. The Information Commissioner can be contacted at: Information Commissioner’s Office, Wycliffe House, Water Lane, Wilmslow, Cheshire, SK9 5AF, or 0303 123 1113, or <a href="mailto:icocasework@ico.org.uk">icocasework@ico.org.uk</a>. </p>
+   
+   <p>Any complaint to the Information Commissioner is without prejudice to your right to seek redress through the courts.</p>
+   
+   <h2 class="govuk-heading-l">Changes to this notice</h2>
+   
+   <p>We may change this privacy notice. When we make changes to this notice, the ‘last updated’ date at the bottom of this page will also change. Any changes to this privacy notice will apply to you and your data immediately. If these changes affect how your personal data is processed, GDS will take reasonable steps to make sure you know.</p>
+    <p>
```
Comment (lfdebrux) [inline docs/views/privacy-policy.html]: I don't think this `<p>` should be here?

```suggestion
```

**Comment by lfdebrux** [comment]
Do we want to change the URL and link in the footer to be 'Privacy notice' to match the new `h1`? We should at least update the `pageTitle` on line 4.

**Comment by NoraGDS** [comment]
@lfdebrux > Do we want to change the URL and link in the footer to be 'Privacy notice' to match the new `h1`? We should at least update the `pageTitle` on line 4.

Yes, please. Thanks for noticing. Could the footer be changed to 'Privacy'? And the URL to 'privacy-notice' please?

**Review by joelanman — APPROVED** [review]

**Comment by lfdebrux** [comment]
> @lfdebrux > Do we want to change the URL and link in the footer to be 'Privacy notice' to match the new `h1`? We should at least update the `pageTitle` on line 4.
> 
> Yes, please. Thanks for noticing. Could the footer be changed to 'Privacy'? And the URL to 'privacy-notice' please?

Done. I've also added a redirect so that the old URL still works.

**Pull request merged**